scripts/build.sh

```diff
@@ -4,6 +4,10 @@ cd cpp
 g++ wordcount_baseline.cpp -std=c++11 -o wordcount_baseline -O3
 g++ wordcount.cpp -std=c++11 -o wordcount -O3
 
+cd ../elixir
+# Elixir has to run the script to compile it (http://stackoverflow.com/questions/35722248/the-command-elixirc-is-compiling-and-executing-the-code)
+echo "wadus" | elixir elixir/wordcount.ex > /dev/null
```

Comment:
Nope, in fact its not. because there is no binary or whatever created.

The way it is used now, is very bad meta-programming voodoo and does not need any precompilation.

So these 3 lines can be removed for now.

But while I am experimenting locally I already transfered this unidiomatic use of the meta programming capabilities into a proper mix-project which also does create a proper executable (this doesn't give any perf. improvements at all!). But it is much more clean.

But I will only continue that version of mine, when I this got merged and I figured out how to manually controll IO-Streams in a more efficient way ;) StdLib does a good job in keeping manual flushing away from me ;)